proj-sys/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "proj-sys"
-description = "Rust bindings for PROJ v9.0.x"
+description = "Rust bindings for PROJ v9.1.x"
 repository = "https://github.com/georust/proj"
-version = "0.23.1"
+version = "0.24.0"
```

Comment:
Are you planning to immediately release this? It'd be nice to see CI pass first, which is one of the reasons I typically bump the version in a separate PR after all the to-be-released changes have landed.

Reply:
We can…bump the bump to a separate PR, and no plans to do anything til CI passes! I was just being mildly optimistic about any other changes since this is -sys.